src/server/plugins/engine/views/index.html

```diff
@@ -14,6 +17,10 @@
         {{ govukErrorSummary(errors) }}
       {% endif %}
 
+      {% if isStartPage %}
```

Comment:
You're already doing a check in the page controller, but the check is repeated here. What if the two drift?

Earlier, you check if it's a start page or a summary page. But here you're only checking if it's a start page. 

I'd keep the check only in the page controller as you did earlier, but just send a flag rather than the individual checks. e.g. `{% if isStartPage %}` would become `{% if isMissingNotificationEmail %}`. isMissingNotificationEmail would be computed in the controller based on `isStartPage || isSummaryPage`.

Reply:
the reason I have check in page controller is to not waste API calls like you said in other comment.

Yes, that's correct because we have different view for summary and for everything else and same model is used for both summary and other pages. 

I think its more efficient the way its done now. I only need to check for notificationEmail for first page. I don't need to do `{% if isMissingNotificationEmail %}` check for every page. Just for more clarify, I can add `notification email` check to `isStartPage` check but it wouldn't make any difference to functionality. 

Happy to get on a call if needed

Reply:
Discussed over a call. Resolved.